src/apps/weblib/js-api/vizzu.d.ts

```diff
@@ -601,6 +601,15 @@ interface GroupOptions
 	delay?: Duration;
 }
 
+/** Algorithm type for transition in case of the data grouped 
+	differently on the source and target chart (different set of categorical
+	dataseries used).
```

Comment:
/** Type of transition when the categorical series differ on the source and the target chart.